gnovm/stdlibs/math/overflow/overflow.gno

```diff
@@ -48,6 +49,16 @@ So, FEEL FREE to carefully review the code visually.
 
 // Unspecified size, i.e. normal signed int
 
+// Abs get absolute value of an int, returning the result and a ok result indicating whether the operation is safe.
+func Abs(x int) (int, bool) {
+	if _is64Bit() {
```

Comment:
The `_is64Bit` function checks whether the current system is 64-bit. If you store the result of this function as a variable, it seems unnecessary to keep calling this function repeatedly.

```go
var is64Bit = _is64Bit()
```

Additionally, you can simplify that function as follows:

```go
func _is64Bit() bool {
    return ^uint(0) > 0xffffffff
}
```

Reply:
commit 8c33095